source/vistas/core/graphics/feature/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .geometry import FeatureGeometry
```

Comment:
Actually, I don't know if this is all that helpful here. We don't need to import the whole module, just the classes that will be useful. It could go one way or another. I kind of like it how it is.